### PR TITLE
Adafruit NeoPixel mounts for V0

### PR DESCRIPTION
**Pull request by QuackProfit**
* [X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [X] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [X] Submitted STL files are oriented and scaled properly for printing.
  * [X] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
This is a new mod to add mounting support for Adafruit's NeoPixel PCBs on the V0's Y-rail extrusions.

#### How was it tested? 
This mod was tested by myself and another member on the Discord.

### CODE REVIEW

File: printer_mods/QuackProfit/V0-Adafruit-Y_Rails/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* (4) M3x6 SHCS/BHCS - to mount the assemblies to the Y-rail extrusions
+* (4) M3 nuts - to mount the assemblies to the Y-rail extrusions
+* (2) JST-XH connector plug 4-position - to allow for easy disconnection (optional)
+* 22awg wire
+
+## Assembly
+* Solder wires of appropriate length onto the back of each PCB (VDC, GND, DIN, and DOUT)
+* If using JST-XH as disconnects, solder the other ends of the wires to 4-position (male) connectors - the connector should fit inside the body cavity once assembled
+* Mount the PCBs inside the printed bodies using M2x8s
+* Don't install the printed covers as this stage
```
Comment (KiloQubit) [inline printer_mods/QuackProfit/V0-Adafruit-Y_Rails/README.md:27]: Replace `as` with `at`.

File: printer_mods/QuackProfit/V0-Adafruit-Y_Rails/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+* If connecting directly to a compatible Controller you have to chain the data signal between the left and right assemblies
+  * Choose either the left or right assembly to be the first in the chain
+  * The Controller's neopixel data output pin will connect to the DIN of that "first" assembly
+  * The DOUT of the "first assembly" must connect to the DIN of the "second" assembly
+  * VDC and GND of both assemblies must be connected to the Controller's neopixel VDC and GND pins
+* Some Controller boards (e.g. SKR) require a capacitor and resistor - [SKR Neopixel Breakout Board](https://github.com/codiac2600/Neopixel-Breakout-Board) [link1](https://www.printedsolid.com/products/skr-neopixel-breakout-board) [link2](https://sparta3d.ca/products/ldo-skr-neopixel-breakout-board)
+
+## Klipper Config
+* Create a "neopixel" section in printer.cfg and give it a name
+* Refer to your Controller documentation to find the pin name that controlls the neopixel data signal
```
Comment (KiloQubit) [inline printer_mods/QuackProfit/V0-Adafruit-Y_Rails/README.md:54]: Replace `controlls` with `controls`.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Just a few minor corrections below.

File: printer_mods/QuackProfit/V0-Adafruit-Y_Rails/README.md (hunk 1 above)
Comment (QuackProfit) [reply printer_mods/QuackProfit/V0-Adafruit-Y_Rails/README.md]: Fixed.

File: printer_mods/QuackProfit/V0-Adafruit-Y_Rails/README.md (hunk 2 above)
Comment (QuackProfit) [reply printer_mods/QuackProfit/V0-Adafruit-Y_Rails/README.md]: Fixed.

**Comment by QuackProfit** [comment]
Thank you for catching my typos! I need to find a good spell checker add-on for VS Code.

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Pull request merged**